superset/commands/report/execute.py

```diff
@@ -168,15 +168,15 @@ def _get_url(
                 force=force,
                 **kwargs,
             )
-
         # If we need to render dashboard in a specific state, use stateful permalink
-        if dashboard_state := self._report_schedule.extra.get("dashboard"):
+        if (
+            dashboard_state := self._report_schedule.extra.get("dashboard")
+        ) and feature_flag_manager.is_feature_enabled("ALERT_REPORT_TABS"):
```

Comment:
Is this supposed to be an `and`?

Reply:
yes, the logic here is saying that if there is dashboard metadata and the feature flag is enabled, open in a stateful permalink, else open in the normal url without permalinks.

Reply:
What I am saying is that now this is only applied if the `ALERT_REPORT_TABS` is enabled but shouldn't it be applied anyway of there is an `extra` parameter? Is there a reason to tie this specifically to the `ALERT_REPORT_TABS` FF?

Reply:
My rationale for hiding that functionality behind the `ALERT_REPORT_TABS` FF was that if a report was saved with an `extra` parameter, then the FF was turned off, there would no longer be an interface for viewing/updating the currently selected tab. It seemed more intuitive to have the behavior default to ignoring the `extra` parameter altogether while the FF was disabled.

Reply:
If we are confident that extra is only used for this use case, then it's good.